.github/workflows/neptune-contrib.yml

```diff
@@ -0,0 +1,28 @@
+name: neptune-contrib
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          pip install -r test_requirements.txt
+          pip install tox-travis pylintfileheader
+      - name: Build and test
+        run : |
+          # python -m pylint -j 0 -f parseable neptunecontrib tests
+          tox
+      - name: Publish package
+        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```

Comment:
Does this make it publish only if push was tagged?

Reply:
Yup